incubator/hnc/Makefile

```diff
@@ -104,29 +107,6 @@ kubectl: build
 run: build
 	go run ./cmd/manager/main.go --novalidation
 
-# Generate the Krew manifest and put it in manifests/ (creating that dir if it
-# doesn't exist).
-krew-build: krew-tar
-	-mkdir manifests
-	cp hack/krew-hierarchical-namespaces.yaml manifests/krew-hierarchical-namespaces.yaml
-	sed -i 's/^\(\s*sha256\s*:\s*\).*/\1"$(KREW_CKSM)"/' \
-		manifests/krew-hierarchical-namespaces.yaml
-	sed -i 's/^\(\s*version\s*:\s*\).*/\1"$(HNC_IMG_TAG)"/' \
-		manifests/krew-hierarchical-namespaces.yaml
-
-# Make krew archive and put into /hack. IS THIS THE RIGHT WAY TO PACKAGE UP
-# BINARIES FOR MULTIPLE OSES?
-krew-tar: build
-	tar -zcvf bin/kubectl-hns.tar.gz bin/kubectl
-
-# Install kubectl plugin locally using krew.
-krew-install: krew-build
-	kubectl krew install --manifest=manifests/krew-hierarchical-namespaces.yaml --archive=bin/kubectl-hns.tar.gz
-
-# Uninstall kubectl plugin locally using krew.
-krew-uninstall:
-	-kubectl krew uninstall hierarchical-namespaces
-
```

Comment:
What's the difference between the old and the new?

Reply:
`krew-tar` now includes the license file, while `krew-build` replaces more parts of the template file.